src/app/send-analytics-hook.js

```diff
@@ -0,0 +1,22 @@
+import ReactGA from "react-ga";
+
+export default function useSendAnalytics() {
+  return ({ path, category, action }) => {
+    const disableAnalytics = localStorage.getItem("disableAnalytics");
+    if (
+      process.env.NODE_ENV !== "production" ||
+      (disableAnalytics !== "false" && disableAnalytics !== undefined)
```

Comment:
Aren't these both falsy? Could this be simplified to;

```
      if(!disableAnalytics){ }
```

Reply:
Technically yes however I wanted it to be explicit because it's not obvious that `localStorage` values are always strings.